Makefile.common

```diff
@@ -66,6 +66,11 @@ LDFLAGS += -X "github.com/pingcap/tidb/util/versioninfo.TiDBGitHash=$(shell git
 LDFLAGS += -X "github.com/pingcap/tidb/util/versioninfo.TiDBGitBranch=$(shell git rev-parse --abbrev-ref HEAD)"
 LDFLAGS += -X "github.com/pingcap/tidb/util/versioninfo.TiDBEdition=$(TIDB_EDITION)"
 
+EXTENSION_FLAG =
+ifeq ($(shell if [ -a extension/enterprise/.git ]; then echo "true"; fi),true)
+	EXTENSION_FLAG += -X "github.com/pingcap/tidb/util/versioninfo.TiDBEnterpriseExtensionVersion=$(shell cd extension/enterprise && git rev-parse HEAD)"
```

Comment:
Bazel also need to add this info at ```build/print-workspace-status.sh```.

Reply:
BTW if it is necessary, we can create ```build/print-workspace-status-with-plugin.sh```.

Reply:
Add in https://github.com/pingcap/tidb/pull/43095/commits/6a5fc14050e05359e63da85c02d6a8784a77855a